### ISSUE TITLE
Modal is not closed when cancel/verify captcha and shows error on Android 

### ISSUE THREAD

**Issue by mhNirob** [hCaptcha/react-native-hcaptcha#28]
When I click outside captcha modal or finish verifying the captcha images disappear but the transparent black background doesn't. And I get `Maximum call stack size exceeded` error.

Seems like  `captchaRef?.hide()` is not working on android, and it is causing this issue.

Code - 

```
  let captchaRef = useRef(null);

  const onMessage = (event) => {
    if (event && event.nativeEvent.data) {
      if (['cancel', 'error', 'expired'].includes(event.nativeEvent.data)) {
        handleCaptchaSubmit(null);
      } else {
        handleCaptchaSubmit(event.nativeEvent.data);
        setTimeout(() => {
          handleCaptchaSubmit(null);
          setIsChecked((prevIsChecked) => !prevIsChecked);
        }, 12000);
      }
      captchaRef?.hide();
    }
  };

  <ConfirmHcaptcha
    mode
    siteKey={Secrets.CAPTCHA_SITEKEY}
    baseUrl={captchaBaseUrl}
    languageCode="en"
    ref={(_ref) => (captchaRef = _ref)}
    onMessage={onMessage}
  />
```

Tested on: Android Emulator - Pixel 3A

<img width="300" alt="Screenshot_1667306407" src="https://user-images.githubusercontent.com/12656914/199234782-88ac00de-ac28-4dd6-a8bf-227dac65f8ec.png">

https://user-images.githubusercontent.com/12656914/199238114-28acf21f-e02f-4360-be0e-b8cc4165ca47.mov

**Comment by e271828-** [comment]
@mhNirob which Android version is this?

**Comment by mhNirob** [comment]
> 

Android version is 13.0

**Comment by CAMOBAP** [comment]
@mhNirob could you please also share information about your app:
1. react-native version?
2. Are you using expo?

**Comment by mhNirob** [comment]
@CAMOBAP 

1. react native version is - 0.68.2
2. No, I am not using expo

**Comment by beqramo** [comment]
Same issue here too, any help?

**Comment by e271828-** [comment]
Closed in 1.4.2
